src/base/types/FluxSummary.jl

```diff
@@ -0,0 +1,89 @@
+"""
+FluxSummary
+
+A struct used to store summary information about the solution 
+of a constraint based analysis result.
+"""
+struct FluxSummary
+    biomass_fluxes :: OrderedDict{String, Float64}
+    import_fluxes :: OrderedDict{String, Float64}
+    export_fluxes :: OrderedDict{String, Float64}
+    unbounded_fluxes :: OrderedDict{String, Float64}
+end
+
+"""
+    flux_summary(flux_result::Dict{String, Float64}; 
+                exclude_exchanges = false,
+                exchange_prefixes = _constants.exchange_prefixes,
+                biomass_strings = _constants.biomass_strings,
+                exclude_biomass = false,
+                small_flux_bound = 1.0/_constants.default_reaction_bound^2,
+                large_flux_bound = _constants.default_reaction_bound,
+                keep_unbounded = false,
+                )::FluxSummary
+
+Return a `FluxSummary` struct based on the `flux_result` of a constraint based
```

Comment:
First paragraph of the docs:

Summarize a dictionary of fluxes into small, useful representation of the most important information contained. Useful for pretty-printing and quickly exploring the results.

Last paragraph can be slashed, users need to discover that themselves anyway :D